components/stepper/index.css

```diff
@@ -315,3 +283,33 @@ governing permissions and limitations under the License.
   border-end-end-radius: var(--mod-stepper-border-radius, var(--spectrum-stepper-border-radius));
   border-inline-end-width: var(--mod-stepper-border-width, var(--spectrum-stepper-border-width));
 }
+
+@media (forced-colors: active) {
+  .spectrum-Stepper {
+    --highcontrast-stepper-border-color: CanvasText;
+    --highcontrast-stepper-border-color-hover: Highlight;
+    --highcontrast-stepper-border-color-focus: Highlight;
+    --highcontrast-stepper-border-color-focus-hover: Highlight;
+    --highcontrast-stepper-border-color-keyboard-focus: CanvasText;
+
+    &.is-disabled {
+      --highcontrast-stepper-border-color: GrayText;
+      --highcontrast-stepper-buttons-border-width: 2px;
```

Comment:
Should we use the token here? Looks like it's `var(--spectrum-border-width-200)`

Reply:
ah yes good suggestion, I went with using the same custom properties that we're using for the overall border width so if that's modified the WHCM border will still match.